### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 23

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#349] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325617-5b26e116-1a67-446b-b656-92152c80d99d.png)

**Comment by Exyss** [comment]
Interpretiamo il problema come una congiunzione logica tra le  affermazioni delle carte.
La formalizzazione ottenuta è quindi: ¬A ∧ ¬B ∧ B

Tuttavia, solo una carta dice il vero, dunque scegliamo di ragionare per casi ragioniamo per casi:

- Caso 1 - Solo la Prima Carta è vera, dunque otteniamo che -----> ¬A ∧ B ∧ ¬B
- Caso 2 - Solo la Seconda Carta è vera, dunque otteniamo che -----> A ∧ ¬B ∧ ¬B
- Caso 3 - Solo la Terza Carta è vera, dunque otteniamo che: -----> ¬A ∧ ¬B ∧ B

Analizziamo i tre casi:
- Caso 1: Impossibile per incongruenza logica, poiché otteniamo che è sia la carta B, sia che non è la carta B
- Caso 2: Possibile, poiché otteniamo che è la carta A e non è la carta B, dunque la carta con l'asso sarebbe la carta A
- Caso 3: Possibile logicamente, poiché otteniamo che è sia la carta  A, sia la carta B. Tuttavia, per definizione stessa del problema sappiamo che la carta con l'asso è SOLO UNA. Dunque anche questo caso è impossibile.

L'unico caso valido è quindi se la **carta B è l'unica vera**, dunque **l'asso si trova sotto la carta A**
